muse_for_anything/db/models/object_relation_tables.py

```diff
@@ -135,10 +135,10 @@ class OntologyObjectVersionToObject(MODEL, IdMixin):
     __tablename__ = "ObjectVersionToObject"
 
     object_version_source_id: Mapped[int] = mapped_column(
-        ForeignKey("OntologyObjectTypeVersion.id"), nullable=False
+        ForeignKey(OntologyObjectVersion.id), nullable=False
```

Comment:
Oof, that is a bad error in the schema. This will require a migration as well. Good find

Reply:
Was an error I made in the first updates that thankfully caused an error when running MUSE4Anything to be recognized

Reply:
Ok, so this is not a change compared to the main branch. Then lets mark this as resolved.